AppSettingsManagerAPI/AppSettingsManagerApi.Infrastructure/MySql/SettingsContext.cs

```diff
@@ -25,9 +25,9 @@ public SettingsContext(DbContextOptions<SettingsContext> options)
     protected override void OnModelCreating(ModelBuilder modelBuilder)
```

Comment:
I think there should be some Permissions models added, and some new migration files since updates are happening to the database tables